contracts/src/chainlink/solidity/emergency/StarknetValidator.sol

```diff
@@ -22,9 +22,13 @@ interface AggregatorValidatorInterface {
 /// @title StarknetValidator - makes cross chain calls to update the Sequencer Uptime Feed on L2
 contract StarknetValidator is TypeAndVersionInterface, AggregatorValidatorInterface, SimpleWriteAccessController {
   // Config for L1 -> L2 message cost approximation
+  // Message Cost = gasAdjustment * gasEstimate * gasPriceL1Feed / 100
   struct GasConfig {
+    // gas units derived from starknet estimate_message_fee
     uint256 gasEstimate;
```

Comment:
What is the estimate given by `estimate_message_fee`? Please document the approximate value in-line here, and also try to add a default + check to Gauntlet if users use a substantially different value. This value should be pretty static, until our L2 contract changes, or Starknet introduces big gas metering changes.

Reply:
Please also update/add info here: https://github.com/smartcontractkit/chainlink-starknet/tree/develop/docs/emergency-protocol

Reply:
I'll need to deploy on alpha-goerli to test the value. Will do it today

Reply:
The value I got was 17266 units. 
```
tarknet estimate_message_fee --from_address 0x00 --address=0x06f4279f832de1afd94ab79aa1766628d2c1e70bc7f74bfba3335db8e728a7e6 --function update_status --inputs 0x1 123123
```

Reply:
Which comes out to around $3 based on today's gas price

Reply:
I don't think we should add a default to gauntlet because for testing you'd like a negligible fee in order to make it easier to test.

Reply:
Update: I added an error for gasAdjustment if it's below 100

Reply:
> I don't think we should add a default to gauntlet because for testing you'd like a negligible fee in order to make it easier to test.

Wouldn't such a test fail with the incorrect fee?

One of the main use cases for Gauntlet is to save operators from making mistakes and educate them about the operations they are executing. So I'd suggest we add the proper checks in, and then we might add a flag to avoid them so we can tweak execution in tests and while debugging.

For example, we could validate so `gasAdjustment` can't go below 100, unless an explicit `--unsafe-skip-validations` flag is set.